### PR TITLE
Adds Pain Infliction Psionic Power

### PR DESCRIPTION
**Pull request by IrkallaEpsilon**
## About The Pull Request

After somehow loosing track last night and talking to Trilby about mind wizard stuff there was the idea floating in the room that psions could be able to do things with pain. So theres that. Ideas were: Pain Transference, Pain Mirror and Pain Infliction.
This PR solely handles "Pain Infliction" or "powerword_pain" in code.

What it does:

If a psion gets an aggressive grab on a person they can inflict pain upon them expending two psi points. Yup. Thats it.

"But Irkalla what the fuck." 

Well if people have deep psi pools and they didn't powerlevel robustness in round to shit they invested into having said psi pools either by picking cog, cog backgrounds, psi point boosting backgrounds locking them out of more interesting choices or by being a cog heavy role which lacks robustness for grabs. So they can disable a single person they get a grab on. Then they are all out of mana and need a potion.  Considering psy regen is slow this makes them able to disable a single person if they somehow manage to get an aggressive grab on them, or somewhat inconvenience them should they lack the psypool.

Yes, I blatantly ripped the code from another psypower and adjusted it.
Yes, Pain Mirror and Transference are still coming but in separate PRs, need to get a good grasp first on how to reduce a value of person and adding said value to another person.

Additionally: Pain Mirror will be either deep maint cube or xenoarch cube, probably deep maint cube but thats beside the point.

Edit: This is ready, Pain Mirror and Pain Transference will need separate PRs since I need to figure out how to make it work without being IrkallaShitcode (Trademarked)

## Changelog
:cl:
add: Psions can now use "Pain Infliction" or rather "Powerword Pain", still not the all powerful "Powerword Naked" the Faceless seem to know.
/:cl:

### CODE REVIEW

File: code/modules/psionics/psion_powers/healing_powers.dm [hunk 1]
```diff
@@ -20,6 +20,30 @@
...
+		usr.show_message(SPAN_DANGER("You must have an aggressive grab to inflict pain upon somebody!"))
+		return
+
+	if(pay_power_cost(psi_point_cost))
+		if(check_possibility(TRUE, L))
+			usr.visible_message(
+					SPAN_DANGER("[usr] forcefully presses a hand upon [L] in an attempt to inflict pain upon them!"),
+					SPAN_DANGER("You force your hand on [L] expanding your mind and inflicting pain upon them!")
+					)
+			L.AdjustHalLoss(50)
```
Comment (Dimasw99) [inline code/modules/psionics/psion_powers/healing_powers.dm]:
```suggestion
			L.adjustHalLoss(50)
```

**Comment by cdb-is-not-good** [comment]
Cool

**Comment by cdb-is-not-good** [comment]
Also for reference a Taser does like 30

**Comment by Trilbyspaceclone** [comment]
Got no reports or anything that its broken so just merging it

**Pull request merged**